hedera-node/pom.xml

```diff
@@ -366,6 +366,12 @@
       <version>${mockito-junit-jupiter.version}</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.hedera.hashgraph</groupId>
+          <artifactId>test-clients</artifactId>
+          <version>0.10.1-SNAPSHOT</version>
+          <scope>test</scope>
+      </dependency>
```

Comment:
I think that referencing logic from the test-clients is not a good move. If we do need any logic from the tests we should think about adding it in the hedera-node code or if we do not need it anymore we should this dependency altogether